scripts/funcTests/runFuncTests.sh

```diff
@@ -183,7 +183,7 @@ fi
 #run mono test
 TestDir="$DIR/artifacts/NuGet.CommandLine.Test/"
 VsTestConsole="$DIR/artifacts/NuGet.CommandLine.Test/vstest/vstest.console.exe"
-TestResultsTrx="$DIR/build/TestResults/monoonmac.trx"
```

Comment:
It doesn't appear in this message. But, are test results for unit test and functional tests?

Reply:
Yes all test results should be uploaded now